src/pyspextool/extract/override_aperturesigns.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import logging
+
+from pyspextool.extract import config
+from pyspextool.io.check import check_parameter
+from pyspextool.io.check import check_qakeywords
+
+from pyspextool.pyspextoolerror import pySpextoolError
+
+
+def override_aperturesigns(aperture_signs:str,
```

Comment:
Did you intentionally leave out an underscore? Why not override_aperature_signs?

Reply:
Yes, I can show you, but later or tomorrow